csharp/src/Google.Protobuf.Test.TestProtos/Google.Protobuf.Test.TestProtos.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <!--
+    This CodeGen project is kept seperate from the original test project for many reasons.

```

Comment:
nit: still says "CodeGen" after renaming to TestProtos

typo: s/seperate/separate/